temporal/api/cloud/cloudservice/v1/service.proto

```diff
@@ -128,6 +128,13 @@ service CloudService {
         };
     }
 
+    // Remove a region from a namespace
+    rpc RemoveNamespaceRegion (RemoveNamespaceRegionRequest) returns (RemoveNamespaceRegionResponse) {
```

Comment:
I assume listing namespace region(s) is done via some other call?

Reply:
Yes. We have GetRegions and GetRegion APIs